summoner-cli/src/Summoner/Text.hs

```diff
@@ -26,3 +29,23 @@ headToUpper t = case T.uncons t of
 -- | Convert every element of a list into text, and squash the results
 tconcatMap :: (a -> Text) -> [a] -> Text
 tconcatMap f = T.concat . map f
+
+-- | Aligns a list of texts by their columns
+alignTable :: [(Text, Text, Text)] -> [Text]
+alignTable metas = map (formatTriple maxLengths) metas
+    where 
+      maxLengths :: (Max Int, Max Int, Max Int)
+      maxLengths = getMaxLengths metas 
```

Comment:
You can make code a bit cleaner if you unwrap from `Max` here. No need to use in other functions, this `newtype` is used only for calculating max. Like this:

```haskell
maxLengths = mapTriple getMax $ getMaxLengths metas
```